src/you_get/extractors/yizhibo.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+__all__ = ['yizhibo_download']
+
+from ..common import *
+import json
+import time
+
+def yizhibo_download(url, output_dir = '.', merge = True, info_only = False, **kwargs):
+    video_id = url[url.rfind('/')+1:].split(".")[0]
+    json_request_url = 'http://www.yizhibo.com/live/h5api/get_basic_live_info?scid={}'.format(video_id)
+    content = get_content(json_request_url)
+    error = json.loads(content)['result']
+    if (error != 1):
+        raise ValueError("Error : {}".format(error))
+
+    data = json.loads(content)
+    title = data.get('data')['live_title']
+    if (title == ''):
+        title = data.get('data')['nickname']
+    m3u8_url = data.get('data')['play_url']
+    m3u8 = get_content(m3u8_url)
```

Comment:
are you sure to do m3u8 extract? for M3u8 live streaming, m3u8 link content only has current slides of the stream, maybe 1 or 2 mins or less.
could you show the result of real_url? and the video length?